library/constant_time_internal.h

```diff
@@ -133,19 +133,19 @@ unsigned mbedtls_ct_mpi_uint_lt( const mbedtls_mpi_uint x,
  * \brief          Check if one unsigned MPI is less than another in constant
  *                 time.
  *
- * \param X        The left-hand MPI. This must point to an array of limbs
- *                 with the same allocated length as \p Y.
- * \param Y        The right-hand MPI. This must point to an array of limbs
- *                 with the same allocated length as \p X.
- * \param len      The number of limbs in \p X and \p Y.
+ * \param A        The left-hand MPI. This must point to an array of limbs
+ *                 with the same allocated length as \p B.
+ * \param B        The right-hand MPI. This must point to an array of limbs
+ *                 with the same allocated length as \p A.
+ * \param limbs     The number of limbs in \p A and \p B.
```

Comment:
For alignment
```suggestion
 * \param limbs    The number of limbs in \p A and \p B.
```

Reply:
Heh, I think we both spotted the same thing :)